### PR TITLE
Docker version of dcm2niix

### PR DESCRIPTION
**Pull request by alexlicohen**
for those who want the portability...

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:trusty
+MAINTAINER <alexcohen@gmail.com> # feel free to change/adopt
+
+ENV http_proxy 'http://proxy.tch.harvard.edu:3128'
+ENV https_proxy 'http://proxy.tch.harvard.edu:3128'
```
Comment (chrisfilo) [inline Dockerfile]: Is this necessary?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+	cd dcm2niix && mkdir build && cd build && \
+	cmake -DBATCH_VERSION=ON .. && \
+	make && make install
+
+# Get dcm2niix from github and compile (without cmake)
+# RUN cd /tmp && \
+# 	git clone https://github.com/rordenlab/dcm2niix.git && \
+# 	cd dcm2niix/console/ && \
+# 	g++ -O3 -I. main_console.cpp nii_dicom.cpp jpg_0XC3.cpp ujpeg.cpp nifti1_io_core.cpp nii_ortho.cpp nii_dicom_batch.cpp  -o dcm2niix -DmyDisableOpenJPEG -DmyDisableJasper && \
+# 	cp dcm2niix /usr/local/bin/
```
Comment (chrisfilo) [inline Dockerfile]: What's the advantage of having this non-cmake option commented out in the Dockerfile?

File: Dockerfile (hunk 1 above)
Comment (alexlicohen) [inline Dockerfile]: no, I forgot to strip these lines out.

File: Dockerfile (hunk 2 above)
Comment (alexlicohen) [inline Dockerfile]: same here

**Comment by chrisfilo** [comment]
Excellent - thanks!

**Pull request merged**